### DISCUSSION TITLE
Regarding private contributions

### DISCUSSION THREAD

**Discussion by geeky01adarsh** [anuraghazra/github-readme-stats#2911] [Q&A] [answered]
I have deployed a private instance, but still, my private contribution is not showing up(PRs, issues). Please help me if I am doing something wrong.
<img src="https://github-readme-stats-eight-tau-89.vercel.app/api?username=geeky01adarsh&show=reviews&show_icons=true&locale=en&bg_color=30,e96443,904e95&title_color=fff&text_color=fff" alt="Adarsh's GitHub stats" />
I am using:-
```https://github-readme-stats-eight-tau-89.vercel.app/api?username=geeky01adarsh&show=reviews&show_icons=true&locale=en```

**Comment by rickstaa** [comment]
@geeky01adarsh This problem is mostly caused by needing to set the right permissions when creating the Personal Access token. Please follow step 9 of the step-by-step guide in the [README](https://github.com/anuraghazra/github-readme-stats#on-vercel).

> 9. Create a personal access token (PAT) [here](https://github.com/settings/tokens/new) and enable the `repo` and `user` permissions (this allows access to see private repo and user stats).

**Comment by geeky01adarsh** [comment]
I have done it @rickstaa 
![image](https://github.com/anuraghazra/github-readme-stats/assets/74068552/826e673b-5b31-4da9-bcbc-22137e6ecfff)

**Reply by qwerty541** [reply, marked as answer]
Personally i`m using classic access token instead of fine-grained beta and i do not experience such problem. Try to migrate on classic token.

**Reply by geeky01adarsh** [reply]
Thanks a lot. It worked :)
